pokete_classes/providers.py

```diff
@@ -46,9 +77,50 @@ def get_attack(self, fightmap, enem):
             ]
         )[0]
 
+    def greet(self, fightmap):
+        """Outputs a greeting text at the fights start:
+        ARGS:
+            fightmap: fightmap object"""
+        fightmap.outp.outp(f"A wild {self.curr.name} appeared!")
+
+    def handle_defeat(self, fightmap, winner):
+        """Function caleld when the providers current Pokete dies
+        ARGS:
+            fightmap: fightmap object
+            winner: the defeating provider"""
+        return False
```

Comment:
What does this `False` represent? The docs don't mention the return value

Reply:
It represents whether or not a Pokete was actually chosen, but I also updated the doc string.